addons/binding/org.openhab.binding.gardena/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,541 @@
+                <option value="connection_not_changed">Connection not changed</option>
+                <option value="com_board_not_available">COM board not available</option>
+                <option value="slipped">Slipped</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="sourceForNextStart" advanced="true">
+        <item-type>String</item-type>
+        <label>Source For Next Start</label>
+        <description>The source for the next start</description>
+        <state readOnly="true">
+            <options>
+                <option value="no_source">No source</option>
+                <option value="completed_cutting_daily_limit">Completed cutting daily limit</option>
+                <option value="week_timer">Week timer</option>
+                <option value="countdown_timer">Countdown timer</option>
+                <option value="mower_charging">Mower charging</option>
+                <option value="completed_cutting_autotimer">Completed cutting autotimer</option>
+                <option value="undefined">Undefined</option>
+            </options>
+        </state>
+    </channel-type>
+
+    <channel-type id="timestampNextStart">
+        <item-type>DateTime</item-type>
+        <label>Timestamp Next Start</label>
+        <description>Timestamp of the next start</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="overrideEndTime" advanced="true">
+        <item-type>DateTime</item-type>
+        <label>Override End Time</label>
+        <description>Override End Time</description>
+        <state readOnly="true"/>
+    </channel-type>
+
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
+        <label>Temperature</label>
+        <description>Temperature</description>
+        <state pattern="%d °C" readOnly="true" />
+    </channel-type>
+
+    <!-- ############################### Gateway ############################### -->
+    <thing-type id="gateway">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="bridge"/>
+        </supported-bridge-type-refs>
+
+        <label>Gardena Smart Home Gateway</label>
+        <description>Represents a Gardena Smart Home Gateway</description>
+
+        <channel-groups>
+            <channel-group id="gateway" typeId="gateway" />
+        </channel-groups>
+        <properties>
+            <!-- device info -->
+            <property name="manufacturer" />
+            <property name="product" />
+            <property name="serialNumber" />
+            <property name="sgtin" />
+            <property name="version" />
+            <property name="category" />
+        </properties>
+    </thing-type>
+
+    <channel-group-type id="gateway">
+        <label>Gateway Info</label>
+        <description>Gateway Info</description>
+        <channels>
+            <channel id="ip_address" typeId="ipAddress" />
```

Comment:
No, because i can not directly talk to the gateway, it's only a simple property.